examples/flax/xsum.py

```diff
@@ -123,6 +124,7 @@ def eval_metrics(self, batch, logits, labels):
 class GenerateCallback(TrainCallback):
     def __init__(self, *args, **kwargs) -> None:
         super().__init__(*args, **kwargs)
+        self.logger = logging.getLogger(TrainCallback.__name__)
```

Comment:
Why `TrainCallback` and not `GenerateCallback`?

Reply:
Nice catch. Fixed it.